src/ls.js

```diff
@@ -60,7 +62,7 @@ function _ls(options, paths) {
       relName = relName.replace(/\\/g, '/');
     }
     if (options.long) {
-      stat = stat || fs.lstatSync(abs);
+      stat = stat || options.link ? fs.statSync(abs) : fs.lstatSync(abs);
```

Comment:
I think you need to put the ternary in parentheses:
```js
stat = stat || (options.link ? fs.statSync(abs) : fs.lstatSync(abs));
```
Otherwise the condition for the ternary is `stat || options.link`.